ansi/deno.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@cliffy/ansi",
+  "version": "1.0.0-rc.4",
+  "exports": "./mod.ts",
```

Comment:
Can we also export individual modules here as well (e.g. `tty`, `colors`, etc) like is possible with the `denoland/x` version? See https://jsr.io/docs/package-configuration#exports

I've been trying to optimize startup times for my deno CLI's and I've found that importing less code can have a noticeable benefit. Moving to JSR I've noticed an increase in startup time which could be related to the fact I'm not forced to import the top level mod.ts. The same could be said for other packages as well where modules can be used standalone without the `mod.ts` including everything.